site/docs/v1/tech/admin-guide/troubleshooting.adoc

```diff
@@ -275,3 +283,20 @@ The fix is:
 Unfortunately, you can't search for the user by email, username, or other characteristics using the User Search API. If you have the user's Id, which may be available in the error messages, you can use that. Searching by UUID doesn't use Elasticsearch. If you do not have the UUID, you should change to the database search engine to find the user. You can switch back to Elasticsearch after you have corrected the `data` field.
 
 If you have a support contract, please https://account.fusionauth.io/account/support/[open a ticket] and we'll help you out. If not, please back up your FusionAuth database and test the fix thoroughly in a staging environment before applying it to your production instance.
+
+==== Ignore Fields With a Mismatched Data Type
```

Comment:
Side note, this is ok with me. But we should clearly indicate that this is not available in FusionAuth Cloud. It is only available when you are self hosting. 

It is also quite possible that we will never ship https://github.com/FusionAuth/fusionauth-issues/issues/1639. 

Instead we would like to do this instead, which is a better fix. 
https://github.com/FusionAuth/fusionauth-issues/issues/1640

Reply:
Sounds good. I'll make a note of that and add that limit into the Cloud limits seciotn.